bedrock/firefox/templates/firefox/channel.html

```diff
@@ -116,6 +116,11 @@ <h2 class="channel-title channel-title-aurora">{{ high_res_img('firefox/channel/
           {% endtrans %}
         {% endif %}
       </p>
+      {% if l10n_has_tag('channel_aurora_android') %}
+        <p class="info">
+          <a rel="external" href="https://support.mozilla.org/kb/will-firefox-work-my-mobile-device">{{_('Will Firefox work on my mobile device?')}}</a>
```

Comment:
@flodolo - note I had to add one new string to this PR since it was originally opened. Did so behind a tag here.